pkg/core/fixtures/image_relocation/image-manifest-relocated.yaml

```diff
@@ -1,23 +1,23 @@
-manifestVersion: 0.1
-images:
-  "reg.org/user/github.com/knative/build/cmd/creds-init-b5dff24742c5c8ac4673dc991e3f960d11b58efdf751d26c54ec5144c48eef30":
-  "reg.org/user/github.com/knative/build/cmd/git-init-fe0d19e5da3fc9e7da20abc13d032beafcc283358a8325188dced62536a66e54":
-  "reg.org/user/github.com/knative/build/cmd/controller-3981b19105aabf3ed66db38c15407dc7accf026f4f4703d7e0ca7986ffd37d99":
-  "reg.org/user/github.com/knative/build/cmd/webhook-b9a97b7d360e10e540edfc9329e4f1c01832e58bf57d5dddea5c3a664f64bfc6":
-  "reg.org/user/proxyv2:0.8.0":
-  "reg.org/user/github.com/knative/serving/cmd/activator-e83258dd5858c8b1e92dbd413d0857ad2b22a7c4215ed911f256f68e2972f362":
-  "reg.org/user/github.com/knative/serving/cmd/autoscaler-76222399addc02454db9837ea3ff54bae29849168586051a9d0180daa2c1a805":
-  "reg.org/user/github.com/knative/serving/cmd/queue-99c841aa72c2928d1bf333348a848b5afb182715a2a0441da6282c86d4be807e":
-  "reg.org/user/fluentd-elasticsearch-e71ae1e56d9a15bbcf26be734d13b608":
-  "reg.org/user/github.com/knative/serving/cmd/controller-28db335f18cbd2a015fd218b9c7ce30b4366898fa3728a7f6dab6537991de028":
-  "reg.org/user/github.com/knative/serving/cmd/webhook-50ea89c48f8890fbe0cee336fc5cbdadcfe6884afbe5977db5d66892095b397d": "50ea89c48f8890fbe0cee336fc5cbdadcfe6884afbe5977db5d66892095b397d"
-  "reg.org/user/proxy_init:1.0.1":
-  "reg.org/user/proxyv2:1.0.1": "00ea89c48f8890fbe0cee336fc5cbdadcfe6884afbe5977db5d66892095b3970"
-  "reg.org/user/hyperkube:v1.7.6_coreos.0":
-  "reg.org/user/statsd-exporter:v0.6.0":
-  "reg.org/user/galley:1.0.1":
-  "reg.org/user/mixer:1.0.1":
-  "reg.org/user/pilot:1.0.1":
-  "reg.org/user/citadel:1.0.1":
-  "reg.org/user/sidecar_injector:1.0.1":
-  "reg.org/user/riff-init-a53b8b34598fd723044b9fc6d0471a62-33130595cba42999c2a25dccb2d25b9accd5e2f6745dad75702e9e061a762d72":
+Images:
```

Comment:
lowercase `images`

Reply:
@scothis Well spotted. Fixed that with a fourth, one-line commit.